{
  "_annotation": {
    "cve_id": "CVE-2025-24372",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2025/CVE-2025-24372.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:okfn:ckan:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "CCA2343F-FE8E-5BA6-9460-3303AFDD7A46",
                "versionEndExcluding": "2.10.7",
                "vulnerable": true
              },
              {
                "criteria": "cpe:2.3:a:okfn:ckan:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "F332BB78-940F-50D0-9670-2B40FEA31E08",
                "versionEndExcluding": "2.11.2",
                "versionStartIncluding": "2.11.0",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}